libs/auth/scripts/configure-dev-java-card

```diff
@@ -4,6 +4,5 @@ set -euo pipefail
 
 SCRIPTS_DIRECTORY="$(dirname "${BASH_SOURCE[0]}")"
 
-VX_CERT_AUTHORITY_CERT_PATH="${SCRIPTS_DIRECTORY}/../certs/dev/vx-cert-authority-cert.pem" \
-    VX_PRIVATE_KEY_PATH="${SCRIPTS_DIRECTORY}/../certs/dev/vx-private-key.pem" \
+NODE_ENV=development \
```

Comment:
Made some updates to how the various libs/auth scripts use env vars, making the scripts more consistent with each other.

No updates to vxsuite-complete-system necessary. And all scripts can still be called as they were, the only exception being the exact command for initial Java Card configuration for production cards. I document that updated command in a subsequent README-update commit. I'll also update other relevant documentation after I merge this PR and share an update in #vxsuite-prodeng